src/protocol/sort/apply_sort/shuffle.rs

```diff
@@ -175,5 +213,42 @@ mod tests {
             // Shuffled output should have same inputs
             assert_eq!(hashed_output_secret, hashed_input);
         }
+
+        #[tokio::test]
+        async fn shuffle_vec_of_replicated() {
+            let original_match_keys = [
```

Comment:
Use `const` for this sort of thing.

As for the name, isn't this just a bunch of random values in the field?  I don't see any match keys.

Reply:
yeah, it's just a bunch of random values.